### DISCUSSION TITLE
Release workarounds as part of MCT that we remove/obsolete with each major release of MAUI

### DISCUSSION THREAD

**Discussion by PureWeen** [CommunityToolkit/Maui#1003] [New Feature Discussions]
### Description

Not sure if MCT is the right place for this but I figured I'd start a discussion around this to get peoples thoughts. 

I've been collecting workarounds for issues that we aren't going to be able to back port fixes for here https://github.com/PureWeen/ShanedlerSamples. 

My plan is to push these out as a nuget people can install but I figured I'd just start a discussion around if this would be an area we could leverage MCT at all for or if it would just muddy up MCT too much.

**Comment by pictos** [comment]
@PureWeen by default, in terms of your suggestion, here's a place where features can mature and evolve before being promoted to be part of the Maui project. So, if one of those workarounds can mature to be in the .NET Maui, like the Keyboard stuff this is the right place! Like, let's say you want to create an MCTShell and try new features and in the future port it to MAUI. 

Now, if those workarounds will not evolve into a solution that will be part of the .NET MAUI, then we need to discuss them one by one. IMHO, if the workaround is solid it's fine to be here, now if can break in some scenarios then can be difficult to add it here, since we will have to maintain the code to make it work for most of the devs, I would say.

**Reply by TheCodeTraveler** [reply]
Adding on to this, we're understaffed. 

If the .NET MAUI team would like to leverage the Community Toolkit, we'll need someone from .NET MAUI team to join our Core Maintainer Team to support bug fixes on each new feature: https://github.com/CommunityToolkit/Maui/wiki/Core-Team-Tools-and-Responsibilities

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
